flake.nix

```diff
@@ -556,18 +550,16 @@
             nixfmt-check = stdenv.mkDerivation {
               name = "nixfmt-check";
               dontUnpack = true;
+
               buildInputs = [ all-directories-and-files nixfmt ];
               installPhase = ''
                 mkdir $out
                 nixfmt --version
-
-                total_exit_code=0
-                for file in $(find ${all-directories-and-files} -type f -and -name "*.nix"); do
-                  echo "=== $file ==="
-                  nixfmt --check $file || total_exit_code=$?
-                  echo "==="
-                done
-                exit $total_exit_code
+                # note, really can just src with filer by .nix, no need all files 
+                SRC=$(find ${all-directories-and-files} -name "*.nix" -type f | tr "\n" " ")
```

Comment:
```suggestion
                SRC=$(find ${all-directories-and-files} -name "*.nix" -type f -printf "%p ")
```

Reply:
design question, i would prefer formatting to be left to other tools so. cause all tools have their own and how to recall? but most tools output can be bend to some format and then formatted with more or less generic tools

Reply:
ok